package.json

```diff
@@ -29,7 +30,8 @@
     "react-mounter": "^1.1.0",
     "react-page-transitions": "^1.2.0",
     "react-swipeable": "^3.1.0",
-    "tether": "^1.2.0"
+    "tether": "^1.2.0",
+    "util": "^0.10.3"
```

Comment:
Only as sub-dependencies, but due to limitations in 1.3's static analysis, it doesn't include stuff that's being required dynamically by packages.  The work-around is to manually require anything that falls into that category at the top level.